src/inmanta/module.py

```diff
@@ -1527,16 +1527,20 @@ def set(cls, project: "Project", *, clean: bool = True) -> None:
             loader.unload_inmanta_plugins()
         loader.PluginModuleFinder.reset()
 
-    def install_modules(self) -> None:
+    def install_modules(self, *, bypass_module_cache: bool = False, update_dependencies: bool = False) -> None:
```

Comment:
Shouldn't the `update_dependencies` parameter be passed on to something?

Reply:
Very good catch. I expect this is what causes such a slowdown. Fixed in e030985b.

Reply:
I'll report on the new test time in this thread as soon as the build completes.

Reply:
23 minutes, similar to master, nice!